scripts/pod_lib_lint.rb

```diff
@@ -83,6 +88,11 @@ def main(args)
       puts "Still working, running for #{number_of_times_checked / 60}min."
     end
   end
+
+  unless pod_lint_result == true
+    exit(1)
```

Comment:
Instead maybe capture the status code from the system command and just return that?

i.e. `pod_lint_status = $?` above and then just `exit(pod_lint_status)` here?